app/controllers/provider/admin/authentication_providers_controller.rb

```diff
@@ -93,12 +93,13 @@ def authorize_authentication_provider(action = action_name)
   end
 
   UPDATE_PARAMS = %i[client_id client_secret automatically_approve_accounts site
-                     realm skip_ssl_certificate_verification branding_state_event
+                     realm skip_ssl_certificate_verification branding_state
```

Comment:
This controller is for the Admin SSO, the issue is about the developer portal SSO


BTW, You should not modify the parameter in the API, it will break many customer integrations

`branding_state_event` is an action which can take 2 values:

- [brand_as_threescale](https://github.com/3scale/porta/blob/6acea5d7722fb2c3319d12066200019b240b0892/app/models/authentication_provider/github.rb#L23)
- [custom_brand](https://github.com/3scale/porta/blob/6acea5d7722fb2c3319d12066200019b240b0892/app/models/authentication_provider/github.rb#L19)